.github/pull_request_template.md

```diff
@@ -14,13 +14,14 @@ _Replace this with a good description of your changes & reasoning._
 
 ### Detailed test instructions:
 
-1. 
-2. 
-3. 
+1.
+2.
+3.
 
 
 ### Changelog entry
 
-> (Optional) Enter a summary of all changes, which could start with `(Fix|Add|Tweak|Update) - `, in this Pull Request to appear in the changelog if it's accepted.
-> Or remain the "Changelog entry" header with empty content if it's no need to show up in the changelog.
-> Otherwise, the title of Pull Request would be used as the changelog content.
\ No newline at end of file
+> _(Optional)_  
```

Comment:
Now that we're changing this, what do you think about changing it all into comments with `<!--  -->`? And maybe just leave one `>` as a placeholder above or below the comments.

Hopefully, WooRelease won't pick those comments as actual changelogs😰

Reply:
It looks like it will actually take those lines (anything that's not an empty line gets taken): https://github.com/woocommerce/woorelease/blob/56223945225487530e49918d786f7203bc2db667/includes/tools/class-changelog.php#L406-L415

I did consider adding a single entry example, maybe something along the lines of
> Fix - This is an example of an option changelog entry.

Reply:
It could be added at the end below everything:
```
### Changelog entry
* Fix -  

<!-- 
(Optional) …
-->

Reply:
Ah, you're right. Then I guess an example wouldn't be such a good idea too. Might be left as an actual changelog by accident 😓

Reply:
> It could be added at the end below everything:

Hmm. This sounds good too. I tested it and it seems to work:

![image](https://user-images.githubusercontent.com/73110514/123120202-2c91e680-d455-11eb-849f-cceb739f1502.png)

Reply:
Thoughts? https://github.com/woocommerce/google-listings-and-ads/pull/812/commits/63ffaf8acf1bc693c522f86499ad2c237539fdef

Reply:
I think it's best to have those examples as comments too and just have an empty `>` under the changelog entry.

Reply:
OK, moved to the comment in aa02beea0a8165e7f36b6ea5f6f2fb6245036559 (note that technically `>` isn't even required, and can also be `-` or `*`).

Reply:
> note that technically > isn't even required, and can also be - or *

Ah, interesting. OK. Looks good. Let's merge this.

![image](https://user-images.githubusercontent.com/73110514/123234387-f5224900-d4eb-11eb-856c-9e9799b16f9a.png)